src/linkgraph/mcf.cpp

```diff
@@ -494,13 +494,18 @@ MCF1stPass::MCF1stPass(LinkGraphJob &job) : MultiCommodityFlow(job)
 	uint size = job.Size();
 	uint accuracy = job.Settings().accuracy;
 	bool more_loops;
+	std::vector<bool> finished_sources;
```

Comment:
```suggestion
	std::vector<bool> finished_sources(size);
```

would be slightly cleaner?